quasar/main.tf

```diff
@@ -23,3 +23,188 @@ resource "aws_subnet" "subnet-b" {
     Name = "Quasar RDS - 1E"
   }
 }
+
+resource "aws_security_group" "bastion" {
+  name        = "Quasar-Bastion"
+  description = "22 from Everywhere"
+  vpc_id      = "${aws_vpc.vpc.id}"
+
+  tags {
+    Name = "Quasar-Bastion"
```

Comment:
I wonder if there's a good way to take `name` on resources that have it and automatically create a tag... 🤔